pages/button/performance-marks.page.tsx

```diff
@@ -0,0 +1,45 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useState } from 'react';
+import Button from '~components/button';
+import Box from '~components/box';
+import { SpaceBetween, Tabs } from '~components';
+
+export default function ButtonsScenario() {
+  const [loading, setLoading] = useState(true);
+  const [disabled, setDisabled] = useState(false);
+  return (
+    <Box padding="xxl">
+      <h1>Performance marks in buttons</h1>
+      <SpaceBetween size="l">
+        <label>
+          <input type="checkbox" checked={loading} onChange={e => setLoading(e.target.checked)} id="loading" />
+          Loading
+        </label>
+        <label>
+          <input type="checkbox" checked={disabled} onChange={e => setDisabled(e.target.checked)} id="disabled" />
+          Disabled
+        </label>
+        <Button variant="primary" loading={loading} disabled={disabled}>
+          Primary button
+        </Button>
+        <Button>Non-primary button</Button>
+
+        <Tabs
```

Comment:
Would Modal be a good test case as the content is "rendered" but is invisible

Reply:
Good idea! Added in the new revision